swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -272,9 +355,23 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
     }
   }
 
-  private fun deletionTimestamp(workConfiguration: WorkConfiguration): Long =
+  // todo eb clean up name
+  // now + retention period
```

Comment:
Do these need to be cleaned up prior to merge?

Reply:
Good call. I'll remove the todo. Already decided they were fine.